### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by navarroaxel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ jobs:
               repo: context.repo.repo,
               body: `**Welcome**, new contributor!
 
-                Please make sure you're read our [contributing guide](CONTRIBUTING.md) and we look forward to reviewing your Pull request shortly ✨`
+                Please make sure you've read our [contributing guide](CONTRIBUTING.md) and we look forward to reviewing your Pull request shortly ✨`
```
Comment (navarroaxel) [inline README.md:186]: Perhaps this better
```suggestion
                Please make sure you've read our [contributing guide](CONTRIBUTING.md) and we'll look forward to reviewing your Pull request shortly ✨`
```

Comment (TWiStErRob) [reply README.md:186]: "look forward" is already somewhat future, you don't need another "will". You suggestion means something else: "we don't look forward to reviewing now, but we will".

**Review by TWiStErRob — APPROVED** [review]

**Comment by TWiStErRob** [comment]
Try again

**Comment by navarroaxel** [comment]
> Try again

I used the `update branch` button in the GitHub web interface but using the `rebase` strategy. The result wasn't what I was expecting. :man_shrugging:

**Comment by TWiStErRob** [comment]
Yeah, for that button press it was really unexpected then...

**Review by joshmgross — APPROVED** [review]

**Pull request merged**